libcxx/include/__hash_table

```diff
@@ -166,7 +166,12 @@ public:
   }
 #endif
 
-  _LIBCPP_HIDE_FROM_ABI explicit __hash_node(__next_pointer __next, size_t __hash) : _Base(__next), __hash_(__hash) {}
+  template <class _Alloc, class... _Args>
+  _LIBCPP_HIDE_FROM_ABI explicit __hash_node(__next_pointer __next, size_t __hash, _Alloc& __na, _Args&&... __args)
+      : _Base(__next), __hash_(__hash) {
+    allocator_traits<_Alloc>::construct(__na, std::addressof(__get_value()), std::forward<_Args>(__args)...);
+  }
+
   _LIBCPP_HIDE_FROM_ABI ~__hash_node() {}
```

Comment:
Should this be changed to `= default;` as well? (while I'm changing this code)

Reply:
A defaulted definition is possibly deleted because `__node_value_type` can be non-trivially destructible. I haven't check whether there's any code destroying the `__hash_node` as a whole (i.e. calling the destructor). If so, we shouldn't default it.